t3/api/proto/google/api/annotations.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright 2015 Google LLC
```

Comment:
Can you include a README that explains where all these protos came from / how to update them?

Reply:
Yes, definitely!